src/core/public/chrome/README.md

```diff
@@ -0,0 +1,275 @@
+getNavLinks$(): Observable<Array<Readonly<ChromeNavLink>>>;
+
+Get the state of a navlink at this point in time :-
+get(id: string): ChromeNavLink | undefined;
+
+Get the current state of all navlinks :-
+getAll(): Array<Readonly<ChromeNavLink>>;
+
+Check whether or not a navlink exists :-
+has(id: string): boolean;
+
+Remove all navlinks except the one matching the given id :-
+showOnly(id: string): void;
+```
+- How to access
+  ###### Get the current state of all navlinks: 
+  ```ts
+  core.chrome.navLinks.getAll() 
+  ```
+
+  ###### Get the state of a navlink at this point in time: 
+  ```ts
+  core.chrome.navLinks.get()
+  ```
+
+### RecentlyAccessedService : 
+
+- Interface : ChromeRecentlyAccessed
+- Signature : ```recentlyAccessed: ChromeRecentlyAccessed```
+- The Recently viewed items are stored in the browser's local storage.
+- You can go back to the recent search/visualization/dashboard , each item has (link,label,id)
+- Methods :
+
+```typescript
+
+Adds a new item to the recently accessed history :-
+add(link: string, label: string, id: string): void;
+ 
+Gets an Array of the current recently accessed history :-
+get(): ChromeRecentlyAccessedHistoryItem[];
+
+Gets an Observable of the array of recently accessed history :-
+get$(): Observable<ChromeRecentlyAccessedHistoryItem[]>; 
+```
+- How to access
+   ###### Adds a new item to the recently accessed history :
+   ```ts
+   chrome.recentlyAccessed.add('/app/map/1234', 'Map 1234', '1234');
+   ```
+   ###### Gets an Array of the current recently accessed history :
+   ```ts
+    chrome.recentlyAccessed.get().forEach(console.log);;
+   ```
+   
+### DocTitleService : 
+- Interface : ChromeDocTitle
+- Signature : ```docTitle: ChromeDocTitle```
+  ###### - How to change the title of the document
+
+
+   ```ts
+   chrome.docTitle.change('My application title')
+   chrome.docTitle.change(['My application', 'My section'])
+   ```
+### UI :
+###### consists of tsx/scss files && renders UI components from css Library e.g ```<Progress props={props}>```
+
+- Adding/overriding existing css : e.g.
+  ```jsx
+      - Create scss file and define class e.g .osdCustomClass{}
+      - pass className prop to UI component.
+        e.g <HeaderBreadcrumbs className="osdCustomClass"/>;
+  ```
+- UI Components :
+  - HeaderBreadcrumbs : Props
+       ```ts
+       - responsive:boolean Hides extra (above the max) breadcrumbs under a collapsed item as the window gets smaller.
+       
+       - truncate: boolean Forces all breadcrumbs to single line and truncates each breadcrumb to a particular width, except for the last item
+       
+	   - max : Collapses the inner items past the maximum set here into a single ellipses item.
+	   
+	   - breadcrumbs : The array of individual Breadcrumb items
+       ```
+### Custom React Hooks :
+<b> useObservable</b> : React state hook that tracks the latest value of an Observable.
+[More About "React-use" custom hooks library](https://github.com/streamich/react-use)
```

Comment:
Do we expose only the `useObservable` hook from `streamich/react-use`? or have we manually copied over the code from that library?

Reply:
At the moment ,we are only exposing useObservable.  I can see react-use in package json!